js/src/apbct-public-bundle.js

```diff
@@ -2983,6 +2987,7 @@ function sendAjaxCheckingDinamicFormData(form) {
     var visible_fields = {};
     visible_fields[0] = apbct_collect_visible_fields(form);
     apbct_visible_fields_set_cookie( visible_fields );
+    form.append(ctNoCookieConstructHiddenField('hidden'));
```

Comment:
This way is for NoCookie way. What about if cookies is native or alternative?

Reply:
This append resolve NoCookie way in iframe cases. For native or alternative way we could make research.